config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-ci.yaml

```diff
@@ -139,6 +114,7 @@ postsubmits:
       testgrid-dashboards: sig-cluster-lifecycle-cluster-api-provider-aws
       testgrid-tab-name: bazel-ci-e2e-master
```

Comment:
Just opened the full file and still see a couple of bazel names lingering. Should we rename them?

Reply:
Fair enough